metis-indexing/src/main/java/eu/europeana/indexing/fullbean/RdfToFullBeanConverter.java

```diff
@@ -63,15 +66,44 @@ public FullBeanImpl convertRdfToFullBean(RdfWrapper record) {
         europeanaAggregation.map(new EuropeanaAggregationFieldInput()).orElse(null));
     europeanaAggregation.map(EuropeanaAggregationType::getCompleteness).map(LiteralType::getString)
         .map(Integer::parseInt).ifPresent(fullBean::setEuropeanaCompleteness);
-    fullBean.setTimestampCreated(europeanaAggregation.map(EuropeanaAggregationType::getCreated)
-        .map(Created::getString).map(RdfToFullBeanConverter::convertToDate).orElse(null));
-    fullBean.setTimestampUpdated(europeanaAggregation.map(EuropeanaAggregationType::getModified)
-        .map(Modified::getString).map(RdfToFullBeanConverter::convertToDate).orElse(null));
+    fullBean.setTimestampCreated(
+        europeanaAggregation.map(EuropeanaAggregationType::getCreated).map(Created::getString)
+            .map(RdfToFullBeanConverter::convertToDate).orElse(null));
+    fullBean.setTimestampUpdated(
+        europeanaAggregation.map(EuropeanaAggregationType::getModified).map(Modified::getString)
+            .map(RdfToFullBeanConverter::convertToDate).orElse(null));
 
     // Done.
     return fullBean;
   }
 
+  private List<Aggregation> convertAggregations(RdfWrapper record) {
+    final Supplier<List<WebResourceImpl>> webResourcesExtractor = new WebResourcesExtractor(record);
+    //Convert the provider aggregations
+    final List<AggregationImpl> providerAggregations = convertList(record.getProviderAggregations(),
+        new AggregationFieldInput(webResourcesExtractor), false);
+
+    //Compute the referenced web resources of the provider aggregation
+    final List<String> providerWebResourceReferences = Optional.ofNullable(providerAggregations)
```

Comment:
Again: is it not better to put the IDs in a set?

Reply:
Not relevant anymore